### DISCUSSION TITLE
Welcome to cbj_app Discussions!

### DISCUSSION THREAD

**Discussion by guyluz11** [CyBear-Jinni/CBJ_App#299] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Kushalrock** [comment]
Hello There, My name is Kushal. 
I really like the flutter framework and dart language. 
With all the things that I learned, I created an open-source project with the name Routinger.
This repository caught my attention because you were doing some amazing things with IoT and I'd like to know more about it.

**Reply by guyluz11** [reply]
Hello @Kushalrock how are you
Here are some resources that will help you learn more about CyBear Jinni

Our site: [cybearjinni.com](https://cybearjinni.com)
Our Discord server where you can ask questions in the chat and do voice talks  
[<img src="https://cdn.icon-icons.com/icons2/2108/PNG/512/discord_icon_130958.png" height="80">](https://discord.gg/mUXfwUY)

If you have more questions feel free to ask 😁
